### ISSUE TITLE
No link in menu

### ISSUE THREAD

**Issue by MBoberski** [MrApplejuice/wp-autologin-links#24]
**Describe the bug**
After I install and activate I do not see it on top bar .

**Please give the following details when submitting a bug**
 - Wordpress 5.5.1
 - Plugin  1.11.3
 - Chrome 86.0.4240.75
 - Helpful but optional:
    - WP REST Cache  2020.3.0

**Comment by MrApplejuice** [comment]
Thanks for the report! I will investigate!

**Comment by MrApplejuice** [comment]
I just spent some time on this issue @MBoberski . I tested it in my development setup:

![grafik](https://user-images.githubusercontent.com/542105/96631766-0ab8f080-1317-11eb-83f7-5cb2065c2013.png)

As well on my live website:

![grafik](https://user-images.githubusercontent.com/542105/96631809-1ad0d000-1317-11eb-9f50-d086dc51c50a.png)

In Chrome(ium) as well as Firefox. The menu renders in all instances (see screenshot). I also installed the WP REST Cache plugin from: https://wordpress.org/plugins/wp-rest-cache/ and it works:

![grafik](https://user-images.githubusercontent.com/542105/96632124-874bcf00-1317-11eb-9770-9a6952e6ce54.png)

So right now I cannot reproduce the problem. Is there any more information you can provide for me to debug the issue? Otherwise, I will assume that this is not a problem with my plugin. Note: The current version of the WP REST Cache plugin was 2020.3.1 - maybe there was a bug in the previous version that triggered the problem?

**Comment by MBoberski** [comment]
![image](https://user-images.githubusercontent.com/2800036/96638576-a3a03980-1320-11eb-8690-bdf558cb8bb2.png)
I have nothing more, Expij Utils is my plugin mostly with helper functions but I disable id as you see on screenshot

**Comment by MBoberski** [comment]
Also I find the bug :P

![image](https://user-images.githubusercontent.com/2800036/96639796-7fddf300-1322-11eb-82c0-69ee842a254c.png)

Can you spot it? Thats why I do not get menu item :)

**Comment by MrApplejuice** [comment]
Ah that is interesting. That check is in there by design to make it impossible to generate links for a random users of your website as a non-privileged user. Only website admins posses the permissions to modify the login credentials of other accounts, so the menu is reserved for admins only right now (I presume you are trying to access it as non-admin?). One thing I could change in the public codebase is that users - if they already have an autologin link - can access the menu for _themselves_. Would that help your usecase? I am not super-sure what you are trying to accomplish, but right now the plugin seems to work as expected.

**Comment by MBoberski** [comment]
No, I was admin, and even that I do not have top menu item before I made that change. But yes, plugin now works as intended. I will do more searching, revert my change and see I I have profile option to generate link.

**Comment by MrApplejuice** [comment]
Mmmh weird. Okay. If that workaround does work for you then that is fine. I will leave this issue open an mark it as "needs verifcation" to see if someone else has similar issues. Maybe we are able to narrow it down when other people encounter the same problem.
